wordpress-mobile/WordPress-Android
# Issue/10633 material media

Fixes #10633 

This PR moves Media Browser and related screens to Material Theme and adds Dark Theme support. There are some code changes for cosmetic reasons.

At the time, not all screens have Dark Theme support, so the layout will look broken at most of the places.

While refactoring is not in the scope of this PR, since I touched those files, I cleaned up/optimized some of the redundant code as suggested by AS, so do not be alarmed by the size of the PR. I suggest turning on "Hide whitespace changes" option during the review.

There is a huge amount of PRs in this feature that covers all the screens within the app, so I'm requesting reviews from a wide variety of people.

Visual changes are already approved 👍 

![comparison_media](https://user-images.githubusercontent.com/728822/70584486-348d7c80-1b76-11ea-8adc-ebf6b66f7688.png)

To test:
- Confirm visual integrity of media browser, free photo library picker, and media settings screens (see screenshots) in both light and dark themes. The theme can be changed from App Settings.

PR submission checklist:
